app/controllers/articles_controller.rb

```diff
@@ -7,6 +7,8 @@ def show
 
   def index
     @articles = Article.newest
+
+    @article = Article.featured
```

Comment:
I would probably name this `@featured_article` so that it's easier to understand what you are dealing with in the views